src/components/dashboard/Dashboard.js

```diff
@@ -656,6 +632,11 @@ const Dashboard = props => {
     [dispatchScrollEvent, handleScrollEnd],
   )
 
+  const onBalanceLayout = useCallback(
```

Comment:
does it a cross-platform ?  won't it break web / android ?

Reply:
also you have to use state var so the effect will be updated after onLayout

Reply:
We don't need to start the animation on each layout change, only on largeHeader change. I tested web/android/ios, it works

Reply:
ok i'll think bout it at Monday. let's keep this onhold..